src/Sulu/Bundle/AdminBundle/Admin/Routing/Route.php

```diff
@@ -100,13 +100,28 @@ public function getPath(): string
         return $this->path;
     }
 
+    public function getView(): string
+    {
+        return $this->view;
+    }
+
     public function addOption(string $key, $value): self
     {
+        // TODO rename to setOption, since old values will be simply overridden?
```

Comment:
I think we really should do this change, although it is quite a big BC break.

Reply:
ok